### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by kevinemoore**
## Description
Updating the Dockerfile for catalog.

### CODE REVIEW

File: catalog/nginx-web.conf [hunk 1]
```diff
@@ -25,4 +25,9 @@ server {
 		add_header X-Frame-Options DENY;
 		try_files $uri /index.html =404;
 	}
+
+    location = /healthcheck {
```
Comment (dimaryaz) [inline catalog/nginx-web.conf]: Just use `/` for healthchecking in the CFN template?

File: catalog/Dockerfile [hunk 2]
```diff
@@ -25,7 +26,15 @@ RUN npm install
 
 # Install the app
 COPY . .
-COPY config.json.example static/config.json
-COPY federation.json.example static/federation.json
-RUN npm run build:dll
+RUN npm run build
 CMD npm start
```
Comment (dimaryaz) [inline catalog/Dockerfile]: You're running both `npm` and `nginx`??

File: catalog/validate_config.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+import json
+from jsonschema import Draft4Validator
+
+schema = None
+schema_path = "./config-schema.json"
+with open(schema_path, "r") as schema_file:
+    schema = json.loads(schema_file.read())
```
Comment (dimaryaz) [inline catalog/validate_config.py]: `schema = json.load(schema_file)`

File: catalog/validate_config.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+config_path = "./config.json.tmpl"
+with open(config_path, "r") as config_file:
+    catalog_config = json.loads(config_file.read())
```
Comment (dimaryaz) [inline catalog/validate_config.py]: Same here

File: catalog/nginx-web.conf (hunk 1 above)
Comment (kevinemoore) [reply catalog/nginx-web.conf]: I tried that and it didn't work. I'm not sure exactly why.

File: catalog/Dockerfile (hunk 2 above)
Comment (kevinemoore) [reply catalog/Dockerfile]: Good catch!

File: catalog/Dockerfile [hunk 5]
```diff
@@ -18,14 +19,16 @@ EXPOSE 3000
 RUN mkdir /opt/app
```
Comment (dimaryaz) [inline catalog/Dockerfile]: Remove these, too - and all of the install code above besides nginx and gettext.

Comment (dimaryaz) [reply catalog/Dockerfile]: Might be a good idea to switch to a newer ubuntu, too - 16.04 is pretty ancient at this point. (Or use the nginx image?)

**Pull request merged**